### PR TITLE
Add a test for DeprecationReportBody

### PR DESCRIPTION
**Pull request by Elchi3**
Does this make sense? Stil learning how to write tests for the collector. 
Inspired from the `CSPViolationReportBody` test.

### CODE REVIEW

File: custom/tests.yaml [hunk 1]
```diff
@@ -1230,11 +1230,42 @@ api:
...
-    toJSON: return null;
+      // Note: the Content-Security-Policy header is already configured on this website
+      if (!('ReportingObserver' in self)) {
+        return {result: false, message: 'ReportingObserver is not defined'};
+      }
+
+      // Create the observer
+      var observer = new ReportingObserver(
+        function(reports, observer) {
+          console.log(reports)
```
Comment (queengooborg) [inline custom/tests.yaml:1244]: We can probably remove the console logging:

```suggestion
```

File: custom/tests.yaml [hunk 2]
```diff
@@ -1230,11 +1230,42 @@ api:
...
+          types: ["deprecation"],
+          buffered: true,
+        }
+      );
+
+      // Start observing
+      observer.observe();
+
+      // call a deprecated createImageBitmap parameter
+      createImageBitmap(document.getElementById('resource-image-black'), { imageOrientation: 'none'});
```
Comment (queengooborg) [inline custom/tests.yaml:1259]: Do you know when this parameter became deprecated?  Hopefully before Chrome 69?

**Review by queengooborg — CHANGES_REQUESTED** [review]

File: custom/tests.yaml (hunk 2 above)
Comment (Elchi3) [reply custom/tests.yaml:1259]: hmmmm excellent catch... I think it only became deprecated in Chrome 111 https://github.com/chromium/chromium/commit/a615c5b37202356f12a68a930499f3a04571ce8b

Also, this is very specific to Chromiums anyways. Probably, non-Chromiums won't set the same deprecation....

So maybe this whole PR is mood.

**Comment by Elchi3** [comment]
Closing this. Hard to write a cross-engine test for this right now.

**Pull request closed without merging**